cannlytics/cannabis-data-science
# Beginning industry census count for 2023

# Cannabis Industry Census

This pull request introduces the Cannabis Industry Census, which aims to provide comprehensive insights into the cannabis industry by analyzing licenses across various states. We've cleaned up state abbreviations, created visualizations for license counts, both historically and latest, and have provided mapping capabilities to showcase license distributions geographically.

## Key features

- Reading historic and latest license data.
- Hot-fix for mis-coded state data.
- A bar chart showing the number of licenses by state.
- A side-by-side bar chart comparing the historic versus latest license counts by state.
- A map visualizing the geographical distribution of licenses.
- A placeholder for a choropleth map (to be developed) showcasing license densities by state.
- A TODO for using NLP techniques to analyze popular naming conventions for cannabis businesses.

## How to Test

1. Ensure the required data files are present in the ./data/ directory.
2. Run the cannabis_industry_census.py script.
3. Review the generated visualizations and maps.
4. Provide feedback for any improvements or issues detected.

## Additional Notes

The choropleth map generation section is currently commented out and needs further development. There's a planned feature to employ NLP for analyzing popular naming conventions among cannabis businesses.
